### DISCUSSION TITLE
Wiki backups dont backup sidebar settings

### DISCUSSION THREAD

**Discussion by PeachesMLG** [requarks/wiki#7031] [Error / Bug Report]
Not sure if this is a feature or a bug, but if you have a custom navigation sidebar, it would be nice for this to be backed up alongside the wiki pages.

**Comment by fnafox** [comment]
If you need to backup your configuration as well as the content, I suggest maintaining regular database backups.
But I agree it would be nice to have a way to export/backup configuration alongside content.
